assets/src/scripts/wdfn-home/site-summary/index.js

```diff
@@ -0,0 +1,40 @@
+import { select } from 'd3-selection';
+import { link } from '../../lib/d3-redux';
+import { Sites } from '../selectors/wdfn-selector';
+
+const SiteSummaries = (node, store) => {
+    const siteSummaryTmpl = (siteName, siteId, siteType) => {
+      return `<li>
+                <h3>${siteName}</h3>
+                <p class="site-id">${siteId}</p>
+                <p class="site-type">${siteType}</p>
+              </li>`;
+    };
+
+    const populateSiteSummaries = (node, sites) => {
+      const mapContainer = document.getElementById('wdfn-map-container');
```

Comment:
I'm  bothered by passing in node and then not using it to find elements within that node. Eventually, if we carry this to the master branch, I think we should either use D3 for selecting elements or use plain vanilla JS and then subscribe to events using the listen function, https://github.com/usgs/waterdataui/blob/master/assets/src/scripts/lib/d3-redux.js#L73. This function is probably in the wrong place since it doesn't require d3 at all